content/en/docs/guides/development/localization/localization-guide.md

```diff
@@ -77,12 +80,19 @@ The following table describes the key elements and attributes of XLIFF.
 
 | Element/Attribute | Description |
 | --- | --- |
-| `<xliff> - srcLang` | Source language - the code of the language, in which *the text to be translated* is expressed |
+| `<xliff> - srcLang` | Source language - the code of the language, in which *the text to be translated* is expressed. |
 | `<xliff> - trgLang` | Target language - the code of the language, in which *the translated text* is expressed |
 | `<group> - name` | Programming language type - "javascript", "c", "cpp", "x-qml" (for Qt/QML) |
 | `<source>` | Source string - *the text to be translated* |
 | `<target>` | Target string - *the translated text* |
 
+##### notes
```

Comment:
If you use note in the document, please use **{{< note >}} note content {{< /note >}}** style instead of "##### notes".